### ISSUE TITLE
Add jemalloc to the Dockerfile

### ISSUE THREAD

**Issue by wgerlach** [docker-library/cassandra#85]
When I start cassandra I get this warning:
`jemalloc shared library could not be preloaded to speed up memory allocations`

Should maybe a `apt-get -y libjemalloc1` be added to the Dockerfile ?

**Comment by tianon** [comment]
Confirmed, after installing `libjemalloc1` I get the following instead, so that's likely something we should consider adding:
```
INFO  22:43:23 jemalloc seems to be preloaded from /usr/lib/x86_64-linux-gnu/libjemalloc.so.1
```

**Comment by wgerlach** [comment]
I created pull request #86 for this.

**Comment by wgerlach** [comment]
This is now pull request #87 .

**Comment by tianon** [comment]
Fixed via #87. :+1:
